### PR TITLE
fix: improve execution logs

### PR DESCRIPTION
**Pull request by nicufk**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: pkg/executor/client/job.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func (c *JobExecutor) Logs(ctx context.Context, id string) (out chan output.Outp
 
 		for l := range logs {
 			entry, err := output.GetLogEntry(l)
-			if err != nil {
+			if err != nil && entry.Type_ != output.TypeUnknown {
```
Comment (vsukhin) [inline pkg/executor/client/job.go:156]: will we have not nil entry there always?

File: pkg/process/exec.go [hunk 2]
```diff
@@ -25,12 +25,12 @@ func ExecuteInDir(dir string, command string, arguments ...string) (out []byte,
 	cmd.Stderr = buffer
 
 	if err = cmd.Start(); err != nil {
-		return buffer.Bytes(), fmt.Errorf("could not start process: %w", err)
+		return buffer.Bytes(), fmt.Errorf("could not start process with command: %s, exited with code:%d, error: %w\noutput: %s", command, cmd.ProcessState.ExitCode(), err, buffer.String())
```
Comment (vsukhin) [inline pkg/process/exec.go:28]: is ProcessState always not nill?

**Review by vsukhin — APPROVED** [review]
looks good have two minor questions

File: pkg/executor/client/job.go (hunk 1 above)
Comment (nicufk) [reply pkg/executor/client/job.go:156]: GetLogEntry is returning a testkube.ExecutorOutput type not a pointer.

**Pull request merged**